gooddata-sdk/gooddata_sdk/catalog/permission/service.py

```diff
@@ -38,3 +43,23 @@ def put_declarative_permissions(
         self._layout_api.set_workspace_permissions(
             workspace_id=workspace_id, declarative_workspace_permissions=declarative_workspace_permissions.to_api()
         )
+
+    def list_available_assignees(self, workspace_id: str, dashboard_id: str) -> CatalogAvailableAssignees:
```

Comment:
Add short docstring. Get inspired in https://github.com/gooddata/gooddata-python-sdk/blob/1141b0fe5051e31475960f964e0ed2e4f720c241/gooddata-sdk/gooddata_sdk/catalog/workspace/service.py#L171

Reply:
done